lib/msf/core/exploit/remote/kerberos/client/pac.rb

```diff
@@ -52,40 +52,51 @@ def build_pac(opts = {})
               logon_time = opts[:logon_time] || Time.now
               checksum_type = opts[:checksum_type] || Rex::Proto::Kerberos::Crypto::Checksum::RSA_MD5
 
-              logon_info = Rex::Proto::Kerberos::Pac::LogonInfo.new(
+              pac_type = Rex::Proto::Kerberos::Pac::Krb5Pac.new
```

Comment:
Let's add an explicit `require` for this / change zeitwerk config, then we can inline the object instantiation as it was before:

```
              pac_type = Rex::Proto::Kerberos::Pac::Type.new(
                key: opts[:checksum_enc_key],
                buffers: [
                  logon_info,
                  client_info,
                  server_checksum,
                  priv_srv_checksum
                ],
                checksum: checksum_type
              )
```

Reply:
I still stopped it from autoloading and moved it closer to where it's used, lemme know if that's cool with you or if I should spend some time trying to make it work as it did before